backend/src/main/scala/cromwell/backend/async/AsyncBackendJobExecutionActor.scala

```diff
@@ -99,8 +99,8 @@ trait AsyncBackendJobExecutionActor { this: Actor with ActorLogging with SlowJob
     case Finish(FailedNonRetryableExecutionHandle(throwable, returnCode)) =>
       completionPromise.success(JobFailedNonRetryableResponse(jobDescriptor.key, throwable, returnCode))
       context.stop(self)
-    case Finish(FailedRetryableExecutionHandle(throwable, returnCode)) =>
-      completionPromise.success(JobFailedRetryableResponse(jobDescriptor.key, throwable, returnCode))
+    case Finish(FailedRetryableExecutionHandle(throwable, returnCode, retryWithDoubleMemory)) =>
```

Comment:
`retryWithMoreMemory`?

Reply:
Perhaps another producty question for @ruchim. The [JIRA ticket](https://broadworkbench.atlassian.net/browse/BA-5933) title says "more" but the description says "double".